ipfs/blocks.go

```diff
@@ -0,0 +1,15 @@
+package ipfs
+
+import (
+	key "github.com/ipfs/go-ipfs/blocks/key"
+	"github.com/ipfs/go-ipfs/commands"
+)
+
+func BlockstoreHas(ctx commands.Context, hash string) (bool, error) {
+	node, err := ctx.GetNode()
+	if err != nil {
```

Comment:
Is it good practice to return false here if there is an error? Someone might not check the error message and assume it's false.